imagemagick.yaml

```diff
@@ -0,0 +1,24 @@
+name: imagemagick
+config-opts:
+  - --enable-shared
+  - --disable-static
+  - --with-modules
+  - --with-x
+  - --with-threads
+  - --with-magick_plus_plus
+  - --with-gslib
+  - --with-wmf
+  - --with-webp
+  - --with-openexr
+  - --with-rsvg
+  - --with-xml
+  - --with-jbig
+  - --with-openjp2
+sources:
+  # Source0
+  - type: archive
+    url: https://github.com/ImageMagick/ImageMagick6/archive/6.9.10-14.tar.gz
+    sha256: c8bcfbf5a88954bf6f82c0fd098cad7818cf52253219173bea4bbd09603c452e
+post-install:
```

Comment:
Not needed. Those files are not exported.

Reply:
> Not needed. Those files are not exported.

This application heavily depends on external tools, especially ImageMagick. It uses not only `convert`, but also less known programs, such as `animate`.
```
#define CMD_GIF_PLAY "animate \"%s\" &"
```

Reply:
Bilal's comment is about installing README, AUTHORS and other files that are not exported by flatpak.